packages/key-management/test/InMemoryKeyAgent.test.ts

```diff
@@ -1,5 +1,13 @@
 import * as Crypto from '@cardano-sdk/crypto';
-import { AddressType, GroupedAddress, InMemoryKeyAgent, KeyRole, SerializableInMemoryKeyAgentData, util } from '../src';
+import {
```

Comment:
In order to have confidence in these changes for our use case, let's update [SharedWallet/simpleTx](https://github.com/input-output-hk/cardano-js-sdk/blob/master/packages/e2e/test/wallet_epoch_0/SharedWallet/simpleTx.test.ts#L18) e2e test to build the script with 1854 purpose keys and sign with those keys as well

Reply:
Uodated on 271064a